src/util/domains.py

```diff
@@ -354,6 +354,7 @@ def adapt_domains(cr, model, old, new, adapter=None, skip_inherit=(), force_adap
                 # this view is already wrong, we don't change it
                 _logger.warning("Skipping domain adpatation for invalid view (id=%s):\n%s", view_id, e.msg)
                 continue
+            _logger.exception("Failing domain adpatation of invalid view (id=%s)", view_id)
```

Comment:
There was a typo here and the warning above: `adpatation`
Sill I'm not sure if we want to log the whole error... the view is already broken, we are not supposed to fix it. We fix views that are correct but do not work due to something done during the upgrade.